src/android/orb.yml

```diff
@@ -238,6 +244,9 @@ commands:
             else
                 echo "We only publish artifacts if the build is from a tag, a PR or from develop and trunk branches."
                 echo "Skipping publishing artifacts!"
+                echo ""
+                echo "If this build was triggered from a commit that is part of an open PR, it's possible it started before the PR was opened."
+                echo "In that case, simply restart the workflow an CircleCI will notice the PR and not halt here."
```

Comment:
I verified this behavior (but not using this branch) by:

1. Pushing a branch and waiting for its build to start. [It did not publish to S3](https://app.circleci.com/pipelines/github/wordpress-mobile/WordPress-Utils-Android/307/workflows/8ca46a8d-b84d-4669-a9a0-0b516aab68b1/jobs/672).
2. Opening a PR and restarting the workflow for that same commit. [It did publish to S3](https://app.circleci.com/pipelines/github/wordpress-mobile/WordPress-Utils-Android/307/workflows/cee33080-86e2-4ec3-9e65-1a0f470bf4da/jobs/673)